zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/RecordExporter.java

```diff
@@ -52,18 +54,25 @@ boolean export() {
       return true;
     }
 
+    final ValueType valueType = typedEvent.getValueType();
+    exporterMetrics.exportingLatency(
```

Comment:
:wrench: Shouldn't we record the latency _after_ it is exported? Or do you really want to measure the time between writing until _starting_ to export?

Reply:
Also, this would measure repeatedly when exporting is retried. More reason to move this to _after_ the record was successfully exported.

Reply:
:thought_balloon: Calculating latency until the start of the export makes sense to me. But we have to consider retries as well. So may be calculate it before the start, but only record after it is successfully exported?

Reply:
Yes this was intended. It is similar to the overall processing metric/ panel (in our dashboard)

I can build then something like this 


![2024-08-20_13-38](https://github.com/user-attachments/assets/8c8bda8f-69a9-49e3-a3b6-7a8ead5413e8)

This allows me to better understand where time is spent, especially when I have the export time separately.

Reply:
> But we have to consider retries as well. So may be calculate it before the start, but only record after it is successfully exported?